utils/types/priority.go

```diff
@@ -108,3 +108,19 @@ func (p *Priority) Scan(value interface{}) error {
 func (p *Priority) Value() (driver.Value, error) {
 	return string(*p), nil
 }
+
+func ScoreToPriority(score float32) Priority {
```

Comment:
That makes sense! Thanks! I would still do Negligible < 1 though. I doubt there is any < 0.1 scores out there.